release-please-config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "packages": {
+    "solutions/landing-zone": {
+      "package-name": "landing-zone",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+    },
+    "solutions/guardrails": {
+      "package-name": "guardrails",
+      "changelog-path": "CHANGELOG.md",
+      "tag-separator": "/",
+      "prerelease": true
+      
+    },
+    "solutions/landing-zone-v2": {
+      "package-name": "landing-zone-v2",
```

Comment:
I think the package-name also needs the full path so it's included in the tag (for it to work with kpt naming convention)
i.e. `"package-name": "solutions/landing-zone-v2"`

same thing for all other packages